editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3422,7 +3422,7 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	animation_hb->add_child(memnew(VSeparator));
 	animation_hb->hide();
 
-	key_loc_button = memnew(Button("loc"));
+	key_loc_button = memnew(Button("Location"));
```

Comment:
That and further changes to the CanvasItemEditor will likely mess it up by making the buttons way too wide.
Maybe we can replace those by icons? @djrm

Reply:
@letheed submited a the icons, so i guess he can simply put the names here, and wait for the new icons to merge into godot.

Reply:
Since we'll have icons I'll just drop the Button changes.